composer.json

```diff
@@ -18,7 +18,7 @@
     ],
     "homepage": "https://codeception.com/",
     "require": {
-        "php": "^8.0",
+        "php": "^8",
```

Comment:
Maybe use "^8.0|^8.1" so we don't match 8.2 that might not be compatible. Better to be explicit than to be surprised later ;)

Reply:
@NicolasCARPi I agree, I have changed the value.